package.json

```diff
@@ -1,66 +1,55 @@
 {
-  "name": "demo-store",
+  "name": "okendo-hydrogen-sandbox",
   "private": true,
-  "version": "0.0.0",
+  "sideEffects": false,
+  "version": "1.0.1",
   "scripts": {
-    "build": "npm run build:css && shopify hydrogen build",
-    "build:css": "postcss styles --base styles --dir app/styles --env production",
-    "dev": "npm run build:css && concurrently -g --kill-others-on-fail -r npm:dev:css \"shopify hydrogen dev\"",
-    "dev:css": "postcss styles --base styles --dir app/styles -w",
-    "preview": "npm run build && shopify hydrogen preview",
+    "build": "sed -i \"s/<your-okendo-subscriber-id>/$OKENDO_SUBSCRIBER_ID/g\" app/root.tsx && shopify hydrogen build",
+    "dev": "shopify hydrogen dev --worker --codegen",
+    "preview": "npm run build && shopify hydrogen preview --worker",
     "lint": "eslint --no-error-on-unmatched-pattern --ext .js,.ts,.jsx,.tsx .",
-    "format": "prettier --write --ignore-unknown .",
-    "format:check": "prettier --check --ignore-unknown .",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "codegen": "shopify hydrogen codegen"
   },
   "prettier": "@shopify/prettier-config",
   "dependencies": {
-    "@headlessui/react": "^1.7.2",
-    "@okendo/shopify-hydrogen": "^2.0.0",
-    "@remix-run/react": "1.14.0",
-    "@shopify/cli": "3.29.0",
-    "@shopify/cli-hydrogen": "^4.0.9",
-    "@shopify/hydrogen": "^2023.1.6",
-    "@shopify/remix-oxygen": "^1.0.4",
-    "clsx": "^1.2.1",
-    "concurrently": "^7.5.0",
-    "cross-env": "^7.0.3",
+    "@okendo/shopify-hydrogen": "file:okendo-shopify-hydrogen-2.0.3.tgz",
```

Comment:
We'll want to change this when the new version launches please

Reply:
Yep, the commit is marked WIP for this reason.